src/core/radio/radio.cpp

```diff
@@ -33,6 +33,58 @@
 
 namespace ot {
 
+namespace {
+#if OPENTHREAD_CONFIG_RADIO_LINK_IEEE_802_15_4_ENABLE
+constexpr Mac::ShortAddress kDefaultShortAddress = 0xfffe;
```

Comment:
Can we directly use the constant `Mac::kShortAddrInvalid `？

Reply:
Done.